issues/disconnect-between-business-and-it.md

```diff
@@ -0,0 +1,56 @@
+---
+slug: disconnect-between-business-and-it
+aspect: culture
+stages:
+- development
+short_description: Modern companies sometimes forget to focus more on the long term IT innovations. They make critical financial decisions without the correct IT-business insight.
+tags:
+- business
+- goal
+- management
+keywords:
+- business
+- innovation
+- finance
+- Kovarus
+- investment
+- budget
+---
+# Disconnect Between Business and IT
+
+## What Is Disconnect Between Business and IT
```

Comment:
I am not sure about headlines. Is that correct `What Is Disconnect Between Business and IT`? Shouldn't it be `What Is Disconnection Between Business and IT` or `What Means Disconnection Between Business and IT`?

Reply:
I think I just missed the article. But "disconnect between business and IT" is a phrase, if I'm correct.

Reply:
Ok, thanks for the explanation. :)